d2common/d2fileformats/d2font/font.go

```diff
@@ -35,12 +35,22 @@ func (fg *fontGlyph) setHeight(h int) {
 	fg.height = h
 }
 
+// GetSize returns glyph's size
+func (fg *fontGlyph) GetSize() (w, h int) {
+	return fg.width, fg.height
+}
+
+// GetFrameIndex returns glyph's frame
+func (fg *fontGlyph) GetFrameIndex() int {
+	return fg.frame
+}
```

Comment:
can you change these to `Size` and `FrameIndex` instead?

Reply:
okey

Reply:
done